string robot_name
float64 time_taken
float64 shortest_distance
---
